Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -260,131 +270,175 @@ private void ContentGrid_ManipulationCompleted(object sender, ManipulationComple
 
             _leftCommandTransform.TranslateX = 0;
             _rightCommandTransform.TranslateX = 0;
-            _leftCommandPanel.Opacity = 1;
-            _rightCommandPanel.Opacity = 1;
+            _commandContainer.Opacity = 0; // How to animate this?
```

Comment:
You can use the animation library to animate it back to 0: _commandContainer.Fade(value: 0f).Start();